openquake/hme/core/core.py

```diff
@@ -223,7 +236,15 @@ def load_inputs(cfg: dict) -> Tuple[GeoDataFrame]:
         Configuration for the evaluations, such as that parsed from the YAML
         config file.
     """
-    rupture_gdf = load_ruptures_from_ssm(cfg)
+
+    rupture_file = cfg["input"]["ssm"]["rupture_file"]
```

Comment:
This fails with a KeyError if it isn't there.  There needs to be a try/except block.